src/App.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import ViewList from './Pages/ViewList';
+import AddItem from './Pages/AddItem';
+import Navbar from './components/Navbar';
+import Home from './Pages/Home';
+import { Route, Switch, BrowserRouter, Link } from 'react-router-dom';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <BrowserRouter>
+          <Navbar />
```

Comment:
I think the links are supposed to be at the bottom.

"Links are present and persistent at the bottom of the app: one for the “list” view, the other for the “add an item” view"

Reply:
I just saw the notes. Maybe this is not as easy as it seems

Reply:
I was able to get the links to the bottom, but commented out the code in the App.css file. If necessary, I can move it to the bottom!